apps/omg_watcher/lib/omg_watcher/block_getter.ex

```diff
@@ -72,6 +72,9 @@ defmodule OMG.Watcher.BlockGetter do
     maximum_block_withholding_time_ms = Application.fetch_env!(:omg_watcher, :maximum_block_withholding_time_ms)
     maximum_number_of_unapplied_blocks = Application.fetch_env!(:omg_watcher, :maximum_number_of_unapplied_blocks)
 
+    # TODO rethink posible solutions see issue #724
+    # wait for ExitProcessor stand up
```

Comment:
let's have a more detailed synopsis of the breaking scenario here, what happens if we don't wait